src/components/Login/LocalLogin.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+// import Transition from '../Transition';
+import Button from '../Common/Button';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+// import { useToasts } from 'react-toast-notifications';
+import axios from 'axios';
+
+const messages = defineMessages({
+  email: 'Email Address',
+  password: 'Password',
+  validationemailrequired: 'Not a valid email address',
+  validationpasswordrequired: 'Password required',
+  loginerror: 'Something went wrong when trying to sign in',
+  loggingin: 'Logging in...',
+  login: 'Login',
+  goback: 'Go back',
+});
+
+interface LocalLoginProps {
+  goBack: () => void;
+  revalidate: () => void;
+}
+
+const LocalLogin: React.FC<LocalLoginProps> = ({ goBack, revalidate }) => {
+  const intl = useIntl();
+  const [loginError, setLoginError] = useState<string | null>(null);
+
+  const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+      .email()
+      .required(intl.formatMessage(messages.validationemailrequired)),
+    password: Yup.string().required(
+      intl.formatMessage(messages.validationpasswordrequired)
+    ),
+  });
+
+  return (
+    <Formik
+      initialValues={{
+        email: '',
+        password: '',
+      }}
+      validationSchema={LoginSchema}
+      onSubmit={async (values) => {
+        try {
+          await axios.post('/api/v1/auth/local', {
+            email: values.email,
+            password: values.password,
+          });
+        } catch (e) {
+          setLoginError(intl.formatMessage(messages.loginerror));
+        } finally {
+          revalidate();
+        }
+      }}
+    >
+      {({
+        errors,
+        touched,
+        isSubmitting,
+        // values,
+        isValid,
+        // setFieldValue,
+      }) => {
+        return (
+          <>
+            <Form>
+              <div className="mt-6 sm:mt-5 sm:border-t sm:border-gray-800">
```

Comment:
The margin top here is not necessary as the parent container already has sufficient padding for the form.